flask_multipass_cern.py

```diff
@@ -16,9 +17,24 @@
 from flask_multipass.group import Group
 from flask_multipass.identity import IdentityProvider
 from flask_multipass.providers.authlib import AuthlibAuthProvider, _authlib_oauth
+from requests.adapters import HTTPAdapter
+from requests.exceptions import HTTPError
+from urllib3 import Retry
 
+from indico.core.logger import Logger
```

Comment:
Please don't import any indico code here. This library is not only used in indico!

You could make the logger name configurable with a default such as `multipass.cern`. Then just use stdlib `logging.getLogger(...)` to get that logger.